client/box/box-card.view.html

```diff
@@ -1,7 +1,7 @@
 <md-card>
   <md-card-title>
     <md-card-title-text>
-      <span class="md-headline">{{box.name}}</span>
+      <a href="#/box/{{box.id}}"><span class="md-headline">{{box.name}}</span></a>
```

Comment:
Could we make it so that the entire box card is a link to the box, rather than just the box title? The title is currently blue and underlined, which looks a bit strange.